woocommerce/woocommerce-gateway-stripe
# Filter out payment methods which do not support current currency

fixes #1739

# Testing instructions

Enable UPE feature flag. 
```sql
INSERT INTO `wp_options` (`option_id`, `option_name`, `option_value`, `autoload`) VALUES (NULL, '_wcstripe_feature_upe', 'yes', 'no') ON DUPLICATE KEY UPDATE `option_value`=VALUES(`option_value`);
```

Enable UPE in Stripe settings and check many payment methods.
Set store currency to not EUR.
Try to checkout with UPE
only Card method should be shown and no errors.
-------------------
